MantidQt/CustomInterfaces/src/MantidEVWorker.cpp

```diff
@@ -156,12 +156,21 @@ bool MantidEVWorker::loadAndConvertToMD(
       IAlgorithm_sptr alg = AlgorithmManager::Instance().create("Load");
       alg->setProperty("Filename", file_name);
       alg->setProperty("OutputWorkspace", ev_ws_name);
-      alg->setProperty("Precount", true);
+      alg->setProperty("FilterByTofMin", 1000.0);
+      alg->setProperty("FilterByTofMax", 16666.0);
       alg->setProperty("LoadMonitors", true);
 
       if (!alg->execute())
         return false;
 
+      alg = AlgorithmManager::Instance().create("FilterBadPulses");
+      alg->setProperty("InputWorkspace", ev_ws_name);
+      alg->setProperty("OutputWorkspace", ev_ws_name);
+      alg->setProperty("LowerCutoff", 95.0);
```

Comment:
New wisdom on this is to set the value to something significantly lower so you measure during beam coming online. The value POWGEN uses is 25.